src/ConnectedMode/ServerSentEvents/SSESessionManager.cs

```diff
@@ -27,16 +27,9 @@
 
 namespace SonarLint.VisualStudio.ConnectedMode.ServerSentEvents
 {
-    /// <summary>
-    /// Reacts to project changes and opens/closes Server Sent Events sessions
-    /// </summary>
-    internal interface ISSESessionManager : IDisposable
```

Comment:
I think you need to make `SSESessionManager : IDisposable` now (one of the code smells).